### PR TITLE
Rewrite limelight setup script to a module

### PR DESCRIPTION
**Pull request by gizmo98**
Based on this script:
https://github.com/stsfin/RetropieLimelightInstaller/blob/master/limelightconfig.sh

I cannot test this module finally because i have no geforce to connect to. Is it necessary to have an own section for limelight? Could put it under ports or somewhere else.

Also see: petrockblog#721

### CODE REVIEW

File: scriptmodules/supplementary/limelight.sh [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    #Install limelight simple theme
+    mkdir /etc/emulationstation/themes/simple/limelight
+    mkdir /etc/emulationstation/themes/simple/limelight/art
+    cp $md_inst/theme.xml /etc/emulationstation/themes/simple/limelight/
+    cp $md_inst/limelight.png /etc/emulationstation/themes/simple/limelight/art/
+    cp $md_inst/limelight_art.png /etc/emulationstation/themes/simple/limelight/art/
+    cp $md_inst/limelight_art_blur.png /etc/emulationstation/themes/simple/limelight/art/
+
+    #Run limelight configuration
+pushd $md_inst
```
Comment (joolswills) [inline scriptmodules/supplementary/limelight.sh]: indentation

File: scriptmodules/supplementary/limelight.sh [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    clear
+    echo -e "\nDiscovering GeForce PC:s, when found you can press ctrl+c to stop the search, or it will take a long time \n"
+    #discover IP-addresses of Geforce pc:s
+    java -jar limelight.jar discover
+    echo -e "\n"
```
Comment (joolswills) [inline scriptmodules/supplementary/limelight.sh]: wouldn't just echo suffice ?

first echo can probably lost the first \n and just be a normal echo.

**Comment by joolswills** [comment]
ill merge the two requests and fix up styling, but please double check indentation etc

**Comment by stsfin** [comment]
Hi just to answer gizmos question, the original version of the script installed limelight under ports. The reason why I made the installer to make it as its own console was because it created all the different resolution settings and the configurator and thus the ports became cluttered.  This not really an issue if there is only a configurator script and a "run limelight" script, and you set the resolution during the configuration. This would anyhow be smarter since the 1080p 60FPS wont run without overclockin the h.264 freg atleast to the ~350 mark. (im using 500 for a higher bitrate).

**Comment by taalas** [comment]
Hi and thanks for creating the supplementary module for limelight!

Some minor things I discovered while testing this:

- line 115 in limelight.sh points to the "ports" instead of the "limelight" directory in roms. I am guessing this is just a copy/paste error...

- the emulationstation script for configuring limelight calls "retropie_packages.sh limelight configure" without sudo and won't work (at least on my system)

Apart from that it works really well!

**Comment by stsfin** [comment]
Have you used the newest version?

**Comment by stsfin** [comment]
If possible Limelight should be build on retropie during install, this would ensure the newest working version.

**Comment by taalas** [comment]
I agree, the points I mentioned are not connected to Limelight itself but rather to the way it is currently embedded into RetroPie though.

**Comment by gizmo98** [comment]
I have seen taalas has already done a PR for the wrong file path. I will add sudo as well.

**Comment by taalas** [comment]
Yes, I wasn't sure if you wanted the sudo in there or rather another approach for the settings so I left that out in my PR ;)

**Pull request merged**